DabraHalleloEndaAbunaSamuel/DHAS013.xml

```diff
@@ -0,0 +1,135 @@
+                        <idno>DHAS-013</idno>                    
+                        </msIdentifier>
+                    <msContents>
+                       <summary/>
+                       <msItem xml:id="ms_i1">
+                           
+                           <title type="complete" ref="LIT3505PaulineEpis">Pauline Epistles</title>
+                         <textLang mainLang="gez"/>
+                         </msItem>                       
+                        
+                        </msContents>
+                   <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>                          
+                                           <material key="parchment"/>
+                                       </support>
+                                       <extent>                                          
+                                                       <measure unit="leaf" quantity="171">171</measure>                                  
+                                                       <dimensions type="outer" unit="mm">
+                                                           <height cert="low">250</height>
+                                                           <width cert="low">220</width>
+                                                           <depth cert="low">90</depth>
+                                                           
+                                           </dimensions>
+                                       </extent>   
+                            <foliation>Foliated; the last leaves after <locus target="#170v"/> are paginated, by mistake.</foliation>
+                            <condition key="deficient">The manuscript is worn. The back board is broken. 
+                                The leather cover is preserved only on the inner sides of the boards. 
+                                Strong discolouration along the margins on <locus target="#3v"/> and <locus target="#172v"/>.
+                            </condition>
+                            
+                         </supportDesc>
+            
+                         <layoutDesc>
+                             <layout columns="2">                                 
+                             </layout>
+                         </layoutDesc>
+                         
+                     </objectDesc>
+                       <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                               <seg type="script">18th-century script.</seg>
+                               <seg type="ink">Black; red.</seg> 
+                               <desc> Careful, regular; by a well-trained scribe (similar to the hand of 
+                                   <persName ref="PRS12076ZawaldaMaryam">Zawalda Māryām</persName>).</desc>
```

Comment:
This sounds to me like it is safe to say that it was not Zawalda Māryām's hand. Perhaps it would be better to say: "(possibly the hand of <persName ref="PRS12076ZawaldaMaryam">Zawalda Māryām</persName>)" if that is what was intended.

Reply:
I do not say that it was not Zawalda Maryam's hand, neither I want to say that this was his hand. :) I only want to record the first optical impression, that their hands look similar. The reasons for this may be several, but this require a closer look.